apps/dashboard/test/models/allowlist_policy_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'test_helper'
+
+class AllowlistPolicyTest < ActiveSupport::TestCase
+
+  test "permitted? should return true if a permitted path is passed" do
+    permitted_path = "/somebody/home/dir"
+
+    cfg = Configuration.allowlist_paths
+    cfg.stubs(:allowlist_paths).returns([permitted_path])
+    allowlist = AllowlistPolicy.new(cfg.allowlist_paths)
+
+    assert allowlist.permitted?(permitted_path)
+  end
```

Comment:
Looking at this a second time, there are no tests for the configuration singleton, so maybe let's incorporate that class into these tests using `with_modified_environment` instead of `Configuration.stubs`. 

So this test case would turn into this:
```ruby
  test "permitted? should return true if a permitted path is passed" do
    with_modified_environment({OOD_ALLOWLIST_PATH: "/somebody/home/dir"}) do
      allowlist = AllowlistPolicy.new(Configuration.allowlist_paths)
      assert allowlist.permitted?(permitted_path)
    end
  end
```